contracts/MidnightLabs.sol

```diff
@@ -136,16 +160,21 @@ contract MidnightLabs is ERC1155Supply, Ownable, Pausable {
     /**
      * @notice When the contract is paused, all token transfers are prevented in case of emergency.
      */
-    function _beforeTokenTransfer(address operator, address from, address to, uint256[] memory ids, uint256[] memory amounts, bytes memory data)
-        internal
-        whenNotPaused
-        override
-    {
+    function _beforeTokenTransfer(
+        address operator,
+        address from,
+        address to,
+        uint256[] memory ids,
+        uint256[] memory amounts,
+        bytes memory data
+    ) internal override whenNotPaused {
         super._beforeTokenTransfer(operator, from, to, ids, amounts, data);
     }
 
+    // I dont think this contract will ever hold funds. Currently, this contracy
+    // cant even accept eth, so I feel this function is not needed.
     function withdraw() external onlyOwner {
```

Comment:
sorry about the other changes, my linter automatically committed them. This was my only comment